atomic_reactor/util.py

```diff
@@ -34,6 +35,28 @@
 from requests.utils import guess_json_utf
 
 logger = logging.getLogger(__name__)
+Output = namedtuple('Output', ['file', 'metadata'])
+
+
+class KojiUploadLogger(object):
```

Comment:
Maybe `koji_util.py` is a better home?

Reply:
Whoops, that's actually what @twaugh had suggested as well, I'd moved it from being in two plugins to util instead of koji_util, will relocate that now.